heartbeat/beater/heartbeat.go

```diff
@@ -336,6 +337,15 @@ func makeESClient(cfg *conf.C, attempts int, wait time.Duration) (*eslegclient.C
 		wait,
 	)
 
+	// Overriding the default ES request timeout:
+	// Higher values of timeouts cannot be applied on the SAAS Service
+	// where we are running in tight loops and want the next succesive check to be run for a given monitor
+	// within the next scheduled interval which could be 1m or 3m
+	timeout := int64((10 * time.Second).Seconds())
```

Comment:
There were at least two other options:

- Create a new parameter in makeEsClient accepting the timeout config
- Make this change in MakeESLoader

Although I added this change in makeESClient for two reasons:

1. We are adding the timeout closer to where the ES connection is created
2. This connection just is used to load monitors' state

-